src/main/java/no/sikt/nva/pubchannels/handler/PublicationChannelClient.java

```diff
@@ -1,26 +1,5 @@
 package no.sikt.nva.pubchannels.handler;
 
-import java.util.Map;
-import no.sikt.nva.pubchannels.channelregistry.ChannelType;
-import no.sikt.nva.pubchannels.channelregistry.model.create.ChannelRegistryCreateJournalRequest;
-import no.sikt.nva.pubchannels.channelregistry.model.create.CreateChannelResponse;
-import no.sikt.nva.pubchannels.channelregistry.model.create.ChannelRegistryCreatePublisherRequest;
-import no.sikt.nva.pubchannels.channelregistry.model.create.ChannelRegistryCreateSeriesRequest;
-import no.sikt.nva.pubchannels.handler.search.ThirdPartySearchResponse;
-import nva.commons.apigateway.exceptions.ApiGatewayException;
+public interface PublicationChannelClient extends PublicationChannelSearchCreateClient, PublicationChannelFetchClient {
```

Comment:
Splitting this client to `PublicationChannelFetchClient` and `PublicationChannelSearchCreateClient`. `CachedClient` will implement `PublicationChannelFetchClient`